gallery/src/components/NxList/NxListExamples.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxListSimpleExample from './examples/NxListSimple';
+import NxListClickableExample from './examples/NxListClickable';
+import NxListClickableLinkExample from './examples/NxListClickableLink';
+import NxListBulletedExample from './examples/NxListBulleted';
+import NxListActionsExample from './examples/NxListActions';
+import NxListMultiLineExample from './examples/NxListMultiLine';
+import NxListEmptyExample from './examples/NxListEmpty';
+import NxListErrorExample from './examples/NxListError';
+import NxListLoadingExample from './examples/NxListLoading';
+import NxListDescriptionExample from './examples/NxListDescription';
+import { NxCode } from '@sonatype/react-shared-components';
+
+const NxListSimpleExampleCode = require('./examples/NxListSimple?raw'),
+    NxListClickableExampleCode = require('./examples/NxListClickable?raw'),
+    NxListClickableLinkExampleCode = require('./examples/NxListClickableLink?raw'),
+    NxListBulletedExampleCode = require('./examples/NxListBulleted?raw'),
+    NxListActionsExampleCode = require('./examples/NxListActions?raw'),
+    NxListMultiLineExampleCode = require('./examples/NxListMultiLine?raw'),
+    NxListEmptyExampleCode = require('./examples/NxListEmpty?raw'),
+    NxListErrorExampleCode = require('./examples/NxListError?raw'),
+    NxListLoadingExampleCode = require('./examples/NxListLoading?raw'),
+    NxListDescriptionExampleCode = require('./examples/NxListDescription?raw');
+const NxListsExamples = () =>
+  <>
+    <GalleryExampleTile title="NxList Simple List Example"
```

Comment:
These titles are a bit more verbose than necessary.  "Simple Example" etc would be fine

Reply:
> These titles are a bit more verbose than necessary. "Simple Example" etc would be fine

Changed.
a268b5dcb8330823b7d4106e1ab3b1ca3269a3a4